google-cloud-dns/src/main/java/com/google/cloud/dns/ProjectInfo.java

```diff
@@ -79,45 +79,96 @@
     /**
      * Returns the maximum allowed number of zones in the project.
      */
+    @Deprecated
     public int zones() {
+      return getZones();
+    }
+
+    /**
+     * Returns the maximum allowed number of zones in the project.
+     */
+    public int getZones() {
       return zones;
     }
 
     /**
      * Returns the maximum allowed number of records per {@link RecordSet}.
      */
+    @Deprecated
     public int resourceRecordsPerRrset() {
+      return getResourceRecordsPerRrset();
+    }
+
+    /**
+     * Returns the maximum allowed number of records per {@link RecordSet}.
+     */
+    public int getResourceRecordsPerRrset() {
       return resourceRecordsPerRrset;
     }
 
     /**
      * Returns the maximum allowed number of {@link RecordSet}s to add per {@link
      * ChangeRequest}.
      */
+    @Deprecated
     public int rrsetAdditionsPerChange() {
+      return getRrsetAdditionsPerChange();
+    }
+
+    /**
+     * Returns the maximum allowed number of {@link RecordSet}s to add per {@link
+     * ChangeRequest}.
+     */
+    public int getRrsetAdditionsPerChange() {
       return rrsetAdditionsPerChange;
     }
 
     /**
      * Returns the maximum allowed number of {@link RecordSet}s to delete per {@link
      * ChangeRequest}.
      */
+    @Deprecated
     public int rrsetDeletionsPerChange() {
+      return getRrsetDeletionsPerChange();
+    }
+
+    /**
+     * Returns the maximum allowed number of {@link RecordSet}s to delete per {@link
+     * ChangeRequest}.
+     */
+    public int getRrsetDeletionsPerChange() {
       return rrsetDeletionsPerChange;
     }
 
     /**
      * Returns the maximum allowed number of {@link RecordSet}s per {@link ZoneInfo} in the
      * project.
      */
+    @Deprecated
     public int rrsetsPerZone() {
+      return getRrsetsPerZone();
+    }
+
+    /**
+     * Returns the maximum allowed number of {@link RecordSet}s per {@link ZoneInfo} in the
+     * project.
+     */
+    public int getRrsetsPerZone() {
```

Comment:
I hate suggesting cobol like names, but `Rrsets` doesn't strike me as helpful.  Why not `getResourceRecordSets` ?